### ISSUE TITLE
Upgrading Docker for Mac breaks local cluster

### ISSUE THREAD

**Issue by peterschussheim** [graphcool/prisma#1818]
# Bug Report

## TL;DR

While I am not certain the reason for these problems reported, I was able to resolve them by manually updating my local cluster's `prisma`container's exposed port.

## Summary

Apologies in advance if this is incomplete, however, it may prove useful to others so I am sharing it here.  

I had a local cluster running well for the past week.  Yesterday, `Docker for Mac` notifies me that a new build of docker is available…I install it and when complete, I restart my containers as one would normally do.  As shown in the screenshot, the `prisma` and `mysql` containers throw errors that it cannot create container for these two services

<img src='https://shipusercontent.com/21e1c5fd70b7565d40992bf7023c2fb8/iTerm2002.png' title='iTerm2002.png' width=1128>

I proceeded to dig a little and inspect how the containers are currently configured, using suggestions from @marktani and [prisma docs](https://www.prismagraphql.com/docs/reference/clusters/docker-aira9zama5), and notice that the `prisma` container is not being exposed.  Below is a mockup (didn't save a screenshot) of what I was seeing:

```bash
$ docker ps
CONTAINER ID  IMAGE                       COMMAND                 CREATED            STATUS            PORTS                   NAMES
7210106b6650  prismagraphql/prisma:1.0.0  "/app/bin/single-ser…"  About an hour ago  Up About an hour                          prisma
1c15922e15ba  mysql:5.7                   "docker-entrypoint.s…"  About an hour ago  Up About an hour  0.0.0.0:3306->3306/tcp  prisma-db
```

### Fix

After updating the `prisma` container to expose port `4466` (using [docker networking: exposing and publishing ports](https://docs.docker.com/engine/userguide/networking/#exposing-and-publishing-ports)), I was back in business and all was running well.

 <details>
    <summary>Kitematic</summary>
      If you have `Kitematic`, navigate to the container settings and check 'Hostname/Ports' tab, then ensure 'Configure Ports' section has a line as shown below:
      <img src='https://shipusercontent.com/5122ffb1944bb48abd028c80f3520de4/Kitematic%20%28Beta%29001.png' title='Kitematic (Beta)001.png' width=731>
      </br>
      Verify this worked as intended: `docker ps` should return something similar to the following:
      
   
    $ docker ps
    CONTAINER ID  IMAGE                       COMMAND                 CREATED            STATUS            PORTS                   NAMES
    7210106b6650  prismagraphql/prisma:1.0.0  "/app/bin/single-ser…"  About an hour ago  Up About an hour  0.0.0.0:4466->4466/tcp  prisma
    1c15922e15ba  mysql:5.7                   "docker-entrypoint.s…"  About an hour ago  Up About an hour  0.0.0.0:3306->3306/tcp  prisma-db
   
</details>

## Resources

- [docker networking: exposing and publishing ports](https://docs.docker.com/engine/userguide/networking/#exposing-and-publishing-ports)
- [slack thread](https://graphcool.slack.com/archives/C0MQJ62NL/p1517603496000644)

**Comment by peterschussheim** [comment]
Resolved by downgrading local `docker for mac` version to stable channel.
